docs/supported-technologies.asciidoc

```diff
@@ -0,0 +1,26 @@
+[[supported-technologies]]
+== Supported Technologies
+
+The Elastic APM RUM JavaScript agent will automatically instrument the following to give you detailed performance metrics of your web page
+
+* Page load metrics
+* Load time of Static Assets
+* API requests (XMLHttpRequest and Fetch)
+
+The agent uses the https://developer.mozilla.org/en-US/docs/Web/API/Navigation_timing_API[Navigation Timing API] and [Resource Timing API] available in the browsers to instrument the page load performance and static assets load times.
+
+The agent uses <<custom-transactions, Custom Transactions API>> to instrument all the outgoing XHR and Fetch API requests from the web page to the backend servers and also adds support for <<distributed-tracing-guide, Distributed Tracing>>.
```

Comment:
I think we should say something like: `The agent automatically captures all the outgoing http requests, by instrumenting both XHR and Fetch APIs ....`

Reply:
Should probably also break distributed tracing into its own sentence to avoid a run on sentence